content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -279,31 +282,44 @@ The following table describes each available provider:
 </tbody>
 </table>
 
-Each provider supports multiple keys - the keys are tried in order for decryption, and if the provider
-is the first provider, the first key is used for encryption.
```

Comment:
This content is removed?

Reply:
I'll move it to appear earlier in the page.